onnxscript/irbuilder.py

```diff
@@ -6,14 +6,16 @@
 import io
 import logging
 import warnings
+from typing import Optional, Sequence
 
 import onnx
-from onnx import helper
+from onnx import ValueInfoProto, helper
 from onnx.defs import onnx_opset_version
 
 import onnxscript
 from onnxscript import type_annotation as ta
 from onnxscript import values
+from onnxscript.onnx_types import ONNXType
```

Comment:
nit: Use onnx_types.ONNXType

Reply:
Let me leave it as-is for now. I think the current file-organization need to be rationalized and improved.